templates/parachain/runtime/src/genesis_config_presets.rs

```diff
@@ -0,0 +1,165 @@
+use cumulus_primitives_core::ParaId;
+use sp_std::prelude::*;
+
+pub use sp_consensus_aura::sr25519::AuthorityId as AuraId;
+
+use crate::{AccountId, SessionKeys, Signature, EXISTENTIAL_DEPOSIT};
+use serde_json::Value;
+use sp_core::{sr25519, Pair, Public};
+use sp_runtime::traits::{IdentifyAccount, Verify};
+#[cfg(not(feature = "std"))]
+use sp_std::alloc::format;
+
+/// Preset configuration name for a local testnet environment.
+pub const PRESET_LOCAL_TESTNET: &str = "local_testnet";
+
+/// Preset configuration name for a development environment.
+pub const PRESET_DEVELOPMENT: &str = "development";
```

Comment:
One annoying little detail about this is that if you don't put `chian_type: Dev` in the chain-spec, PJS apps won't show the accounts to you. There is a CLI flag for that two, but seem annoying to repeat `development` many times. 

```
./chain-spec-builder --chain-type development --preset development etc etc 
```

We either create a ticket in PJS apps to change this (prob better), or we think about how we can improve it on our side. Maybe a `chain-spec-builder --dev` that does both? combined with running the chain in `--dev` it sounds like a good dev ex to me. 

```
# compile wasm
# ./chain-spec-builder --runtime wasm --dev > spec.json
# ./omni-node --chain spec.json --dev
```

cc @gupnik @shawntabrizi

Reply:
> `chain-spec-builder --dev`

This would assume that presets' naming is standardized. Every runtime shall provide something in order to make flags in some other tool work correctly. This is probably OK, question is we want to go in that direction.

Reply:
To me it sounds reasonable that these are separate flags. You could have multiple runtime presets for development, all of them with chainType "Dev".

I was thinking how one could do the pjs route. I think currently pjs calls the `system_chainType` RPC which gets its data directly from the chain spec. If we don't want to use the chain-spec property for this, we would need another easy way to tell that we are a dev chain.

Reply:
We can keep separate flags, `--dev` can be convenient shortcut.

Reply:
yeah, let's keep all things configurable in the super-user mode, but one `--dev` that makes some assumptions is a good nice to have.